src/libraries/System.Security.Cryptography/src/System/Security/Cryptography/CapiHelper.Unix.cs

```diff
@@ -94,7 +91,7 @@ public static HashAlgorithmName NameOrOidToHashAlgorithmName(string nameOrOid)
             return null;
         }
 
-        public static HashAlgorithmName? OidToHashAlgorithmName(string oid)
+        internal static HashAlgorithmName? OidToHashAlgorithmName(string oid)
```

Comment:
I think this can be replaced entirely by `HashAlgorithmName.FromOid` now. Don't have to do it here, just leaving a note as something one of us can do afterward.